docs/release-notes/8.16.asciidoc

```diff
@@ -0,0 +1,118 @@
+[[release-notes-header-8.16.0]]
+== 8.16
+
+[discrete]
+[[release-notes-8.16.0]]
+=== 8.16.0
+
+[discrete]
+[[known-issue-8.16.0]]
+==== Known issues
+
+// tag::known-issue-189676[]
+[discrete]
+.Tags appear in Elastic AI Assistant's responses
+[%collapsible]
+====
+*Details* +
+On August 1, 2024, it was discovered that Elastic AI Assistant's responses when using Bedrock Sonnet 3.5 may include `<antThinking>` tags, for example `<search_quality_reflection>` ({kibana-issue}189676[#189676]).
+
+====
+// end::known-issue-189676[]
+
```

Comment:
I still need to add [known issues](https://github.com/elastic/security-docs/issues/5264#issuecomment-2255420571) for the manual rule run feature.
cc: @nkhristinin @approksiu for visiblity

Reply:
Added in [d5e25e3](https://github.com/elastic/security-docs/pull/5977/commits/d5e25e33e1dfdc51e65f9d3735830772a248f1cf).